packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -1,33 +1,201 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
-import type { ProgressSlots, ProgressState } from './Progress.types';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import type { ProgressState, ProgressSlots } from './Progress.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const progressClassNames: SlotClassNames<ProgressSlots> = {
   root: 'fui-Progress',
-  // TODO: add class names for all slots on ProgressSlots.
-  // Should be of the form `<slotName>: 'fui-Progress__<slotName>`
+  bar: 'fui-Progress__bar',
+  track: 'fui-Progress__track',
+  label: 'fui-Progress__label',
+  description: 'fui-Progress__description',
+};
+
+// If the percentComplete is near 0, don't animate it.
+// This prevents animations on reset to 0 scenarios.
+const ZERO_THRESHOLD = 0.01;
+
+// Internal CSS vars
+export const progressCssVars = {
+  percentageCssVar: '--fui-Progress--percentage',
+  transitionCssVar: '--fui-Progress--transition',
+};
+
+const barThicknessValues = {
+  medium: '2px',
+  large: '4px',
+};
+
+const IndeterminateProgress = {
```

Comment:
```suggestion
const indeterminateProgress = {
```

Let's follow naming conventions for variables